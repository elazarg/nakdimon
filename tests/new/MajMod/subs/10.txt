קָתֶרִין שׁוֹתֶה אַלְכּוֹהוֹל כָּל אַחֵר הַצָּהֳרַיִים. בֶּטַח קָשֶׁה לְךָ לְהִישָּׁאֵר פיכחת בחתונות. לֹא קָשֶׁה לִי לִהְיוֹת פיכחת. זֶה כישרון. אָמְנָם לגמתי קְצָת פֹּה וְשָׁם בִּזְמַן הַהֵירָיוֹן, ולמרבה הַמַּזָּל למילי לֹא נִגְרַם נֶזֶק, אֲבָל לֹא כְּדַאי לָהּ לְהִסְתַּכֵּן. עַל מָה אֶת מְדַבֶּרֶת? לַחַיִּים סמוקות, פָּנִים נפוחות, שדיים מְלֵאִים. קָתֶרִין בְּהֵירָיוֹן. ומצאתי שָׁלוֹשׁ בְּדִיקוֹת חִיּוּבִיּוֹת כשסידרתי בִּשְׁבִילָהּ אֶת המיחזור. אֶת גַּם פורצת לטלפונים? הִיא לֹא. הִיא אָמְרָה לְךָ, נָכוֹן? ג'יין, אֲנִי נשבעת לְךָ, לֹא כְּדַאי לְךָ לְהִתְקָרֵב לְזֶה. אֲנִי חַיֶּיבֶת לְהַצִּיעַ לחברתי עֶזְרָה בפיכחות. בָּרוּר שֶׁיֵּשׁ לָהּ בְּעָיָה. -בָּזֶה אֶת לֹא טוֹעֶה. הַתִּינוֹק שֶׁל שפ, נָכוֹן? מָה שֶׁנִּדְמֶה לְךָ שֶׁאַתְּ יוֹדַעַת... אַף אֶחָד לֹא צָרִיךְ לָדַעַת, בְּעֶצֶם. בָּרוּר? אִמָּא שֶׁלִּי אוֹמֶרֶת שֶׁהוּא הַבֵּן שֶׁל בַּת דּוֹדָהּ שֶׁל שריל. וְהוּא יָבִיא אֶת הַטַּבַּעַת. אַבָּא שֶׁלּוֹ הִשְׁתַּמֵּשׁ פַּעַם בְּכַרְטִיס אַשְׁרַאי שֶׁל הַחֶבְרָה כְּדֵי לִקְנוֹת ג'קוזי, וְהוּא הסתבך. אֵיךְ אִמָּא שֶׁלְּךָ יוֹדַעַת? אִמָּא שֶׁלִּי יוֹדַעַת הַכּוֹל עַל כּוּלָּם. טוֹב, אֲנִי אתחתן אִיתּוֹ. לאוליב יֵשׁ חָבֵר! -תַּפְסִיקִי. אִמָּא שֶׁלִּי סִיפְּרָה לִי עוֹד סוֹד. אֲבָל אָסוּר לִי לְסַפֵּר. נוּ, תְּסַפְּרִי לִי. אָסוּר לִי, אֲבָל אִם תפריחי לוֹ נְשִׁיקָה... אֲנִי אֲסַפֵּר לְךָ.