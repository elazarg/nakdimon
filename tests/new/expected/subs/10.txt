קָתְרִין שׁוֹתָה אַלְכּוֹהוֹל כָּל אַחַר הַצָּהֳרַיִים.
בֶּטַח קָשֶׁה לָךְ לְהִישָּׁאֵר פִּיכַּחַת בַּחֲתוּנּוֹת.
לֹא קָשֶׁה לִי לִהְיוֹת פִּיכַּחַת. זֶה כִּישָּׁרוֹן.
אָמְנָם לָגַמְתִּי קְצָת פֹּה וָשָׁם בִּזְמַן הַהֵירָיוֹן,
וּלְמַרְבֵּה הַמַּזָּל לְמִילִי לֹא נִגְרַם נֶזֶק,
אֲבָל לֹא כְּדַאי לָהּ לְהִסְתַּכֵּן.
עַל מָה אַתְּ מְדַבֶּרֶת?
לְחָיַיִם סְמוּקוֹת, פָּנִים נְפוּחוֹת,
שָׂדָיִים מְלֵאִים. קָתְרִין בְּהֵירָיוֹן.
וּמָצָאתִי שָׁלוֹשׁ בְּדִיקוֹת חִיּוּבִיּוֹת
כְּשֶׁסִּידַּרְתִּי בִּשְׁבִילָהּ אֶת הַמִּיחְזוּר.
אַתְּ גַּם פּוֹרֶצֶת לְטֵלֵפוֹנִים?
הִיא לֹא.
הִיא אָמְרָה לָךְ, נָכוֹן?
גֵּ'יְין, אֲנִי נִשְׁבַּעַת לָךְ,
לֹא כְּדַאי לְךָ לְהִתְקָרֵב לְזֶה.
אֲנִי חַיֶּיבֶת לְהַצִּיעַ לַחֲבֶרְתִּי עֶזְרָה בְּפִיכְּחוּת.
בָּרוּר שֶׁיֵּשׁ לָהּ בְּעָיָה.
-בַּזֶּה אַתְּ לֹא טוֹעָה.
הַתִּינוֹק שֶׁל שֶׁפּ, נָכוֹן?
מָה שֶׁנִּדְמֶה לָךְ שֶׁאַתְּ יוֹדַעַת...
אַף אֶחָד לֹא צָרִיךְ לָדַעַת, בְּעֶצֶם.
בָּרוּר?
אִמָּא שֶׁלִּי אוֹמֶרֶת שֶׁהוּא הַבֵּן שֶׁל בַּת דּוֹדָה שֶׁל שְׁרִיל.
וְהוּא יָבִיא אֶת הַטַּבַּעַת.
אַבָּא שֶׁלּוֹ הִשְׁתַּמֵּשׁ פַּעַם
בְּכַרְטִיס אַשְׁרַאי שֶׁל הַחֶבְרָה כְּדֵי לִקְנוֹת גָּ'קוּזִי,
וְהוּא הִסְתַּבֵּךְ.
אֵיךְ אִמָּא שֶׁלָּךְ יוֹדַעַת?
אִמָּא שֶׁלִּי יוֹדַעַת הַכּוֹל עַל כּוּלָּם.
טוֹב, אֲנִי אֶתְחַתֵּן אִיתּוֹ.
לְאוֹלִיב יֵשׁ חָבֵר!
-תַּפְסִיקִי.
אִמָּא שֶׁלִּי סִיפְּרָה לִי עוֹד סוֹד.
אֲבָל אָסוּר לִי לְסַפֵּר.
נוּ, תְּסַפְּרִי לִי.
אָסוּר לִי,
אֲבָל אִם תַּפְרִיחִי לוֹ נְשִׁיקָה...
אֲנִי אֲסַפֵּר לָךְ.
